### PR TITLE
Provisioning in User Defined Private Vnet

### PR DESCRIPTION
**Pull request by abedareNasuni**

### CODE REVIEW

File: provision_nac.sh [hunk 1]
```diff
@@ -184,7 +229,7 @@ run_cognitive_search_indexer(){
     ACS_API_KEY=$2
     ACS_INDEXER_NAME="indexer"
 
-    INDEXER_RUN_STATUS=`curl -d -X POST "https://${ACS_SERVICE_NAME}.search.windows.net/indexers/${ACS_INDEXER_NAME}/run?api-version=2021-04-30-Preview" -H "Content-Type:application/json" -H "api-key:${ACS_API_KEY}"`
+    INDEXER_RUN_STATUS=`curl -d -X POST "https://${ACS_SERVICE_NAME}.search.windows.net/indexers/${ACS_INDEXER_NAME}/run?api-version=2021-04-30-Preview" -H "Content-Type:applicauion/json" -H "api-key:${ACS_API_KEY}"`
```
Comment (psahuNasuni) [inline provision_nac.sh]: Spelling mistake "applications" at line no 232

File: provision_nac.sh [hunk 2]
```diff
@@ -226,14 +272,241 @@ destination_blob_cleanup(){
...
+    done
+    NAC_SUBNETS+="]"	
+    NAC_SUBNETS=$(echo "$NAC_SUBNETS" | sed 's/ //g')
+    DISCOVERY_OUTBOUND_SUBNET=$(echo "$DISCOVERY_OUTBOUND_SUBNET" | sed 's/ //g')
+}
+
 ###### START - EXECUTION ######
 ### GIT_BRANCH_NAME decides the current GitHub branch from Where Code is being executed
-GIT_BRANCH_NAME=""
+GIT_BRANCH_NAME="CTPROJECT-256"
```
Comment (psahuNasuni) [inline provision_nac.sh]: GIT_BRANCH_NAME="CTPROJECT-256"
remove the hardcoded branch name

**Review by psahuNasuni — APPROVED** [review]
Reviewed and code changes suggested.

File: provision_nac.sh (hunk 2 above)
Comment (abedareNasuni) [reply provision_nac.sh]: resolved

**Pull request merged**